components/Team/Team.tsx

```diff
@@ -0,0 +1,59 @@
+import { gql } from '@apollo/client';
+import { useTeamQuery } from '../../src/graphql/types';
+import { useIndexQuery } from '../../src/graphql/types';
+
+interface Props {
+  team_id: string;
+}
+
+gql`
+  query Team($team_id: String!) {
+    Team(team_id: $team_id) {
+      name
+    }
+  }
+`;
+gql`
+  query Index {
+    allTeams {
+      team_id
+      name
+      managers {
+        nickname
+        manager_id
+      }
+      team_logos {
+        url
+      }
+    }
+  }
+`;
+
+const Teams = () => {
+  const { data, loading } = useIndexQuery();
+  console.log(data);
```

Comment:
Let's clean up the console.logs so our console isn't bogged down.

Reply:
cleaned up will be a part of next commit